test/netlist_results.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+cd ..
+
```

Comment:
I don't think we need this `.sh` file. We'll just call the python script directly.